.idea/dataSources.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
This file should not be included as it contains machine-specific data and will mess everyone else's project up

Reply:
I don't know what this file is. It doesn't even show up when I'm looking at my branch in Pycharm so how would I get rid of it?

Reply:
It's in the .idea folder which does not show up in pycharm so you would have to delete the file in windows explorer, then push the deletion.